server/setup_flow.go

```diff
@@ -0,0 +1,453 @@
+		WithPretext("##### :white_check_mark: Configure the Mattermost app in Jira").
+		WithTitle("Enable development mode.").
+		WithText("Mattermost Jira Cloud integration requires setting your Jira to _development mode_. " +
+			"Enabling the development mode allows you to install apps like Mattermost, from outside the Atlassian Marketplace." +
+			"Complete the following steps, then select **Continue**:\n\n" +
+			"1. Navigate to [**Settings > Apps > Manage Apps**]({{.JiraURL}}/plugins/servlet/upm?source=side_nav_manage_addons).\n" +
+			"2. Select **Settings** at the bottom of the page.\n" +
+			"3. Select **Enable development mode**, then select **Apply**.\n").
+		WithButton(continueButton(stepCloudUploadApp)).
+		WithButton(skipButton(stepCloudInstalledApp)).
+		WithButton(cancelButton)
+}
+
+func (p *Plugin) stepCloudUploadApp() flow.Step {
+	return flow.NewStep(stepCloudUploadApp).
+		WithTitle("Upload the Mattermost app (atlassian-config) to Jira.").
+		WithText("To finish the configuration, create a new app in your Jira instance by following these steps:\n\n" +
+			"1. From [**Settings > Apps > Manage Apps**]({{.JiraURL}}/plugins/servlet/upm?source=side_nav_manage_addons) select **Upload app**.\n" +
+			"2. In the **From this URL field**, enter: `{{.ACURL}}`, then select **Upload**\n" +
+			"3. Wait for the app to install. Once completed, you should see an \"Installed and ready to go!\" message.\n").
+		WithButton(flow.Button{
+			Name:     "Waiting for confirmation...",
+			Color:    flow.ColorDefault,
+			Disabled: true,
+		})
+}
+
+func (p *Plugin) stepCloudInstalledApp() flow.Step {
+	next := func(to flow.Name) func(*flow.Flow) (flow.Name, flow.State, error) {
+		return func(f *flow.Flow) (flow.Name, flow.State, error) {
+			jiraURL := f.State[keyJiraURL]
+			instanceID := types.ID(jiraURL)
+			return to, flow.State{
+				keyConnectURL:        p.GetPluginURL() + "/" + instancePath(routeUserConnect, instanceID),
+				keyWebhookURL:        p.getSubscriptionsWebhookURL(instanceID),
+				keyManageWebhooksURL: cloudManageWebhooksURL(jiraURL),
+			}, nil
+		}
+	}
+	return flow.NewStep(stepCloudInstalledApp).
+		WithTitle("Confirmed.").
+		WithText("You've finished configuring the Mattermost App in Jira. Select **Continue** to connect your user account.").
+		WithButton(flow.Button{
+			Name:    "Continue",
+			Color:   flow.ColorPrimary,
+			OnClick: next(stepConnect),
+		}).
+		WithButton(flow.Button{
+			Name:    "DEBUG Skip to webhook",
+			Color:   flow.ColorWarning,
+			OnClick: next(stepWebhook),
+		}).
+		WithButton(cancelButton)
+}
+
+func (p *Plugin) stepConnect() flow.Step {
+	return flow.NewStep(stepConnect).
+		WithPretext("##### :white_check_mark: Connect your Jira user account").
+		WithText("Go **[here]({{.ConnectURL}})** to connect your account.").
+		WithButton(flow.Button{
+			Name:     "Waiting for confirmation...",
+			Color:    flow.ColorDefault,
+			Disabled: true,
+		})
```

Comment:
0/5 is the button really needed? Would you please share a screenshot on how it looks like?